src/sql/workbench/contrib/dashboard/browser/contents/dashboardWidgetWrapper.component.ts

```diff
@@ -243,48 +243,52 @@ export class DashboardWidgetWrapper extends AngularDisposable implements OnInit
 	private updateTheme(theme: IColorTheme): void {
 		const el = <HTMLElement>this._ref.nativeElement;
 		const headerEl: HTMLElement = this.header.nativeElement;
-		let borderColor = theme.getColor(themeColors.DASHBOARD_BORDER);
-		let backgroundColor = theme.getColor(colors.editorBackground, true);
-		const foregroundColor = theme.getColor(themeColors.SIDE_BAR_FOREGROUND, true);
-		const border = theme.getColor(colors.contrastBorder, true);
+		if (this._config.showHeader === false) {
+			headerEl.style.display = 'none';
```

Comment:
You could probably also just do this in the template. 

`*ngIf="_config.showHeader===false` 

I like that better since it matches how we hide the other elements and actually removes it from the DOM which is better style wise.